src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2668,6 +2668,32 @@ public WindowBlurInfo BlurInfo
             }            
         }
 
+        /// <summary>
+        /// Calculates sizes and positions of views in all layers if the view's layout is required to be updated.
+        /// e.g. RequestLayout() is called for the view.
+        /// Since UpdateLayout calculates sizes and positions of views manually right now, do not abuse calling it.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void UpdateLayout()
+        {
+            LayersChildren?.ForEach(layer =>
+            {
+                if (layer?.LayoutCount > 0)
+                {
+                    layer?.Children?.ForEach(view =>
+                    {
+                        if (view != null)
+                        {
+                            using (var size = GetSize())
```

Comment:
I think you may reuse size by storing it to a local variable to avoid duplicate DALi call and heap allocation for every iteration.